tests/test_roipoly.py

```diff
@@ -55,3 +55,9 @@ def test_get_mean_and_std(roi1, img1, roi1_expected_mask):
     expected_std = np.nanstd(masked_img)
     np.testing.assert_almost_equal(mean, expected_mean)
     np.testing.assert_almost_equal(std, expected_std)
+
+
+def test_get_roi_coordinates(roi1):
+    roi_coordinates = roi1.get_roi_coordinates()
+    expected_roi_coordinates = list(zip(roi1.x, roi1.y))
```

Comment:
This is a bit of a weak test, as it just repeating the function's code. It would be better to expicitely state what the expected ROI coordinates would be